lib/readable.rb

```diff
@@ -1,58 +1,31 @@
 module Readable
   def write(file_path, type)
     if type == 'encrypt'
-      File.write(file_path, "#{encryption}, #{key}, #{date}\n", mode: 'a')
+      File.write("./lib/#{file_path}", "#{encryption}\n", mode: 'a')
     elsif type == 'decrypt'
-      File.write(file_path, "#{message}, #{key}, #{date}", mode: 'a')
+      File.write("./lib/#{file_path}", "#{message}\n", mode: 'a')
     end
   end
 
   def status_message(file_path)
-    "Created '#{File.basename(file_path)}' with the key #{key} and the date #{date}"
+    "Created '#{File.basename("./lib/#{file_path}")}' with the key #{key} and the date #{date}"
   end
 
-  def shift_type(type, info)
-    info = info.flatten
+  def shift_type(type, message, *data)
+    message = message.flatten
     if type == 'encrypt'
-      encrypt(info[0], *info[1], *info[2])
+      encrypt(message[0], *data[0], *data[1])
     elsif type == 'decrypt'
-      decrypt(info[0], *info[1], *info[2])
+      decrypt(message[0], data[0], *data[1])
     end
   end
 
-  def read_message(read_file_path, write_file_path, type)
-    File.foreach(read_file_path) do |line|
+  def read_message(read_file_path, write_file_path, *data, type)
+    File.foreach("./lib/#{read_file_path}") do |line|
       working_line = line.chomp.downcase.split(', ')
```

Comment:
No need for the `.split` on the working line since the arguments that were being split are now received from the command line not the text file.